pkg/upgrade/v1_22_0_test.go

```diff
@@ -0,0 +1,57 @@
+package upgrade
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+
+	v1 "github.com/jaegertracing/jaeger-operator/pkg/apis/jaegertracing/v1"
+)
+
+func TestUpgradeJaegerTagssv1_22_0(t *testing.T) {
+	latestVersion := "1.22.0"
+	opts := v1.NewOptions(map[string]interface{}{
+		"jaeger.tags": "somekey=somevalue",
+	})
+
+	nsn := types.NamespacedName{Name: "my-instance"}
+	existing := v1.NewJaeger(nsn)
+	existing.Status.Version = "1.19.0"
```

Comment:
nit: Probably best to have as most recent version (e.g. `1.21.0`) so does not trigger any other upgrade code.

Reply:
Done!